lib/configParser.js

```diff
@@ -1,50 +1,170 @@
-var path = require('path'),
-    glob = require('glob'),
-    log = require('./logger'),
-    helper = require('./util');
-
+"use strict";
+var path = require('path');
+var glob = require('glob');
+var logger = require('./logger');
 // Coffee is required here to enable config files written in coffee-script.
 try {
-  require('coffee-script').register();
-} catch (e) {
-  // Intentionally blank - ignore if coffee-script is not available.
+    require('coffee-script').register();
+}
+catch (e) {
 }
-
 // LiveScript is required here to enable config files written in LiveScript.
 try {
-  require('LiveScript');
-} catch (e) {
-  // Intentionally blank - ignore if LiveScript is not available.
+    require('LiveScript');
 }
-
-var ConfigParser = function() {
-  // Default configuration.
-  this.config_ = {
-    specs: [],
-    multiCapabilities: [],
-    rootElement: 'body',
-    allScriptsTimeout: 11000,
-    getPageTimeout: 10000,
-    params: {},
-    framework: 'jasmine',
-    jasmineNodeOpts: {
-      showColors: true,
-      stackFilter: helper.filterStackTrace,
-      defaultTimeoutInterval: (30 * 1000)
-    },
-    seleniumArgs: [],
-    seleniumSessionId: null,
-    mochaOpts: {
-      ui: 'bdd',
-      reporter: 'list'
-    },
-    chromeDriver: null,
-    configDir: './',
-    plugins: [],
-    skipSourceMapSupport: false
-  };
-};
-
+catch (e) {
+}
+var ConfigParser = (function () {
```

Comment:
are these clang-format changes? if so it would be better in a separate commit to make this review more focused